### DISCUSSION TITLE
Suggestion: Right Click to Close Window

### DISCUSSION THREAD

**Discussion by ShabbyRaccoon** [OpenRCT2/OpenRCT2#21430] [Feature requests] [answered]
OpenTTD has a very useful QOL feature that allows you to enable an option that allows you to close a window by right-clicking it. By default, the setting is disabled but I've had it enabled for so long that I try to right click windows in OpenRCT2. Would it be possible to implement something like that in OpenRCT2?

**Comment by pfroud** [comment]
In OpenTTD, does the window close when you right-click _anywhere_ in the window, even on GUI widgets?

**Reply by ShabbyRaccoon** [reply]
It right clicks if you right-click anywhere in a window, whether it's a GUI element like a button or scroll bar, viewport, or part of the window title.

**Comment by harvito** [comment]
I think I would see this as a useful addition to window management. Questions for implementation, what happens when
- the cursor moves slightly, but stays within, window between press and release?
- I press in one window, move to another, then release?
- I press in one window, move to no window, then release?

#22111 is the pr

**Comment by pfroud** [comment, marked as answer]
A pull request for this feature request was submitted and closed: https://github.com/OpenRCT2/OpenRCT2/pull/22111
